### PR TITLE
feat(User): Change role to roles

### LINKED ISSUE #257: feat(User): Change role to roles
The endpoint ```/api/user/info``` currently returns an object with a role field which is a single string. Change this field to a roles field which is an array of strings. Users can actually have multiple roles so this will allow us to return all the roles a user has.

### PR DESCRIPTION
**Pull request by geoffreykwan**
## Changes
- Changed the ```/api/user/info``` endpoint to return roles string array instead of role string

## Test
- Test with https://github.com/WISE-Community/WISE-Client/pull/1657
- Make sure the ```/api/user/info``` endpoint returns all the roles a user has

Closes #257

### CODE REVIEW

File: src/main/java/org/wise/portal/presentation/web/controllers/user/UserAPIController.java [hunk 1]
```diff
@@ -133,6 +124,26 @@ HashMap<String, Object> getUserInfo(Authentication auth,
...
+      roles.add("trustedAuthor");
+    }
+    if (user.isTeacher()) {
+      roles.add("teacher");
+    }
+    if (user.isStudent()) {
+      roles.add("student");
+    }
+    return roles;
+  }
```
Comment (hirokiterashima) [inline src/main/java/org/wise/portal/presentation/web/controllers/user/UserAPIController.java]: Should this be a method in User object?

**Review by hirokiterashima — COMMENTED** [review]
Functionality works. I'm wondering if getRoles() should be a method in the User class for better encapsulation?

**Comment by geoffreykwan** [comment]
Yea that's a good idea. I've moved the getRoles() function to the User class.

**Review by hirokiterashima — APPROVED** [review]
LGTM

**Pull request merged**